deploy/examples/statefulset-manual-sidecar.yaml

```diff
@@ -0,0 +1,45 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: example-statefulset
+  labels:
+    app: example-app
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: example-app
+  serviceName: example-service
+  template:
+    metadata:
+      labels:
+        app: example-app
+    spec:
+      containers:
+        - name: example-app
+          image: jaegertracing/vertx-create-span:operator-e2e-tests
+          ports:
+            - containerPort: 8080
+              protocol: TCP
+        - name: jaeger-agent
+          image: jaegertracing/jaeger-agent:1.17.0
```

Comment:
What should I put for the image tag in this example? `latest`? `master`? Remove it and just state `image: jaegertracing/jaeger-agent`?

Reply:
1.17.0 is fine. Ideally, our release script would bump this, like this:

https://github.com/jaegertracing/jaeger-operator/blob/deec90def3c674ee3c3656426a1295f04930a3b4/.ci/release.sh#L46-L47

Otherwise, just leave a comment saying that it should (ideally) match the operator's version.